Frontend/Gmfctn/src/app/modules/dashboard/exoft-achievements/exoft-achievements.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import '../../../models/achievement';
+import '../../../models/user';
+import { SaythankModalComponent } from '../../modal-windows/saythank-modal/saythank-modal.component';
+@Component({
```

Comment:
Between imports and any decorator (in this case @Component) should be empty line. Please provide it here.